src/NuGet.Clients/VsExtension/source.extension.vsixmanifest

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="NuGet.0d421874-a3b2-4f67-b53a-ecfce878063b" Version="3.4.0" Language="en-US" Publisher="Microsoft Corporation" />
-    <DisplayName>NuGet Package Manager for Visual Studio 2015</DisplayName>
+    <Identity Id="NuGet.72C5D240-F742-48D4-A0F1-7016671E405B" Version="3.4.0" Language="en-US" Publisher="Microsoft Corporation" />
+    <DisplayName>NuGet Package Manager for Visual Studio 15</DisplayName>
     <Description>A collection of tools to automate the process of downloading, installing, upgrading, configuring, and removing packages from a VS Project.</Description>
     <Icon>Resources/nuget_96.png</Icon>
     <PreviewImage>Resources/nuget_256.png</PreviewImage>
   </Metadata>
   <Installation InstalledByMsi="false" AllUsers="true">
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.VWDExpress" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.VPDExpress" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.VSWinExpress" Version="[14.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.VSWinDesktopExpress" Version="[14.0,]" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="15.0" />
```

Comment:
Should you use `[15.0,16.0)` to start in a more defensive posture? You can always release an updated VSIX that allows 16.x use after you've tested it.
Or perhaps simply specifying 15.0 has the same effect, I don't know.